include/dslash_quda.h

```diff
@@ -532,7 +532,7 @@ namespace quda {
      @param[in] profile The TimeProfile used for profiling the dslash
   */
   void ApplyNdegTwistedCloverPreconditioned(ColorSpinorField &out, const ColorSpinorField &in, const GaugeField &U,
-      const CloverField &C, double a, double b, double c, bool xpay, const ColorSpinorField &x, int parity, bool dagger,
```

Comment:
I think we need the `xpay` argument at this stage to make the choice in the `apply` function between the different ways of instantiating the kernel, would you agree?

Reply:
I had just forgotten to pull it through all the way to the args.

Reply:
yes, you are right...